.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs: # Reusable configuration objects from CircleCI
+  ruby: circleci/ruby@1.0
+jobs:
+  test:
+    docker:
+      - image: cimg/ruby:2.7
+    steps:
+      - checkout
+      - ruby/install-deps # Using the orb to install deps
+      - ruby/rspec-test
```

Comment:
this could be an iteration once this is on circle, but can we report test coverage? as the test are running in a single container it should be not that hard to report back

Reply:
This is just a base CircleCI configuration, there is going to be more steps